docs/source/essentials/normalized-cache.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: Normalized cache
+---
+
+import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
+
+Apollo Android has 2 different kinds of caches: HTTP cache and normalized cache. The HTTP cache is easier to setup but also has more limitations. This page focuses on the normalized cache. If you're looking for a simpler albeit coarser cache, take a look at the [HTTP cache](http-cache).
+
+## Data Normalization:
+
+The normalized cache stores objects by ID.
+
+```graphl
+query BookWithAuthorName {
+  favoriteBook {
+    id
+    title
+    author {
+      id
+      name
+    }
+  }
+}
+
+query AuthorById($id: String!) {
+  author(id: $id) {
+      id
+      name
+    }
+  }
+}
+```
+
+In the above example, requesting the author of your favorite book with the `AuthorById` query will return a result from the cache if you requested your favorite book before. This is because the author is stored only once in the cache.
```

Comment:
What do you think to mention that the selection set of fields should be the same or less than in cache? Otherwise we won;t be able to resolve it.

Reply:
I added a small explanation and a link to https://www.apollographql.com/blog/the-concepts-of-graphql-bc68bd819be3. Since the process of normalization is the same for all platforms, I'd like to focus on the Android specific parts in this page.